WooCommerce/Classes/ViewRelated/Products/SKU Scanner/ProductSKUBarcodeScannerCoordinator.swift

```diff
@@ -0,0 +1,45 @@
+import Experiments
+import UIKit
+
+/// Coordinates navigation for product SKU barcode scanner based on camera permission.
+final class ProductSKUBarcodeScannerCoordinator: Coordinator {
+    var navigationController: UINavigationController
```

Comment:
Nit: Should this variable be private(set) instead?

Reply:
It has to be mutable publicly because of [`Coordinator` protocol](https://github.com/woocommerce/woocommerce-ios/blob/4451a0a327177dc09bc804ee1c48a6272084f277/WooCommerce/Classes/Tools/Coordinator.swift) 😐 

https://github.com/woocommerce/woocommerce-ios/blob/4451a0a327177dc09bc804ee1c48a6272084f277/WooCommerce/Classes/Tools/Coordinator.swift#L8

Reply:
Oh I didn't know about that 😅

Reply:
yea, I don't think the navigation controller needs to be mutable in the protocol. It might be mutable in the first place so that the navigation controller can be set after initialization